### PR TITLE
feat: Using cache to list/get resource

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes https://github.com/kleveross/klever-model-registry/issues/29

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/klever-model-registry/main.go [hunk 1]
```diff
@@ -43,10 +48,12 @@ func gracefulShutdown(closing, done chan struct{}) {
 }
 
 func main() {
-	err := client.InitClient()
-	if err != nil {
+	if err := client.InitClient(stopCh); err != nil {
```
Comment (gaocegege) [inline cmd/klever-model-registry/main.go:51]: When will it stop?

Comment (simon-cj) [reply cmd/klever-model-registry/main.go]: comments are addressed, PTAL

**Comment by simon-cj** [comment]
ping @gaocegege

**Pull request merged**